spec/support/fixtures.rb

```diff
@@ -297,58 +297,79 @@ class Book < Dry::Struct
 
 User = Struct.new(:name, :website, :snippet)
 
-module TestView
-  def self.included(view)
-    view.class_eval do
-      include Hanami::View
+class TestScope < Hanami::View::Scope
+  include Hanami::Helpers
+end
```

Comment:
I think a more realistic way to set this up for testing would be to include the helpers module inside a context class, rather than a scope class. Doing it this way ensures the helpers are available in _any_ aspect of a view rendering, even if a new scope is used inside a template, e.g.

```erb
<h1>This is my view</h1>

<%= scope(:special_scope).render(:some_partial) %>
```

^ In this example, if the helpers were included in the view's scope, they'd no longer be available inside `_some_partial`, because that's being rendered with a `SpecialScope` as its scope, no longer the view's default scope.

By putting the helpers in the context, they're available within both the top-level template, _and_ the `_some_partial`, even when it's rendered with a custom scope.

Happy for you to consider this for a follow-up PR, however, since what you have here is certainly working for the tests as they currently stand!